src/server/helpers/utils.js

```diff
@@ -27,11 +27,22 @@ const Publisher = require('bookbrainz-data').Publisher;
 const Work = require('bookbrainz-data').Work;
 const Promise = require('bluebird');
 
+/**
+ * Returns an API path for interacting with the given entity object
```

Comment:
Entity object -> "Bookshelf entity model" in this comment to be more specific about what's expected